src/models/real-fevr-marketplace.ts

```diff
@@ -0,0 +1,128 @@
+import {Model} from '@base/model';
+import {Web3Connection} from '@base/web3-connection';
+import {Web3ConnectionOptions} from '@interfaces/web3-connection-options';
+import {Deployable} from '@interfaces/deployable';
+import RealFevrMarketplaceJson from '@abi/RealFevrMarketplace.json';
+import {RealFevrMarketplaceMethods} from '@methods/real-fevr-marketplace';
+import {AbiItem} from 'web3-utils';
+import {Errors} from '@interfaces/error-enum';
+import {ERC20} from '@models/erc20';
+import {RealFevrOpener} from '@models/real-fevr-opener';
+import {toSmartContractDecimals} from '@utils/numbers';
+
+export class RealFevrMarketplace extends Model<RealFevrMarketplaceMethods> implements Deployable {
+  constructor(
```

Comment:
please, follow the code style that was implemented previously. Provide arguments on the same line or break _after the first_ argument and never after the `(` as that might cause erroneous reading of the code.

Reply:
done